exercises/poker/src/test/java/PokerTest.java

```diff
@@ -0,0 +1,164 @@
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+
+import jdk.nashorn.internal.ir.annotations.Ignore;
+
+public class PokerTest {
+    @Test
+    public void oneHand() {
+        final String hand = "4S 5S 7H 8D JC";
+        assertEquals(Arrays.asList(hand),new Poker(Arrays.asList(hand)).getBestHands());
+    }
+
+    @Test
+    @Ignore
```

Comment:
You don't have to be sorry, it's my fault for not paying attention to POLICIES doc. Thanks for the comment :)